frontend/redux/selectors.js

```diff
@@ -24,3 +24,22 @@ export const isLoading = state => Object
 
 export const demographics = state => state.demographics.questions;
 export const demographicsOrder = state => state.demographics.order;
+
+export const selectWorkload = function(state) {
+  const images = [];
+  const answers = store.getState().demographics.answers;
+
+  Object.keys(answers).map((i) => {
+    images.push({
+      id: parseInt(i),
+      demographics: answers[i],
+    });
+  });
```

Comment:
There is a little bit of danger doing a `map` here...

Every time you create a new object in a selector, it likely means you are mutating properties on some connected react component, which causes a re-render, (which probably doesn't actually have any changes) and wastes CPU cycles.

You should prefer moving the mutations to the reducers instead, if you need to have the values in a `keyed` and `array` format, you can store both, and select the appropriate one here.